gems/scip-ruby/bin/scip-ruby

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
It would've been nice to have split this up into a different PR, but it seems like busywork right now, so not doing that.